test/resources/v3/program-orders.test.js

```diff
@@ -1,11 +1,14 @@
 const nock = require('nock');
 const RO = require('../../..');
 const fixtures = require('../../fixtures/v3/program-orders.fixtures');
+const { mockConfig } = require('../../test-helpers/mock-config');
+
+RO.config.init(
+  mockConfig({ apiVersion: 'v3', piiServerUrl: null, clientId: 'programTest123', clientSecret: 'itsATestGetUsedToIt' })
```

Comment:
Same

Reply:
Optional: Also, can we leverage the `mockConfig` `clientId` and `clientSecret` props for the — oh, never mind. What a mess. I suppose we need a test helper for creating a Base64 bearer token?

Reply:
Also, to ensure atomicity, `RO.config.init` should go into a `beforeEach` and `afterAll` should convert to a `afterEach`